userv2/user_contact.go

```diff
@@ -0,0 +1,9 @@
+package userv2
+
+// UserContact contains data of user contact.
+type UserContact struct {
+	ID            string `json:"id"`
+	To            string `json:"to"`
+	ContactMethod string `json:"contactMethod"`
```

Comment:
given this value is from a set of fixed values - are there plans to make this a Constant/Enum?